ocs_ci/deployment/aws.py

```diff
@@ -77,6 +77,81 @@ def add_volume(self, size=100):
         logger.info(f'Worker pattern: {worker_pattern}')
         self.create_ebs_volumes(worker_pattern, size)
 
+    def host_network_update(self):
+        """
+        Update security group rules for HostNetwork
+        """
+        cluster_id = get_infra_id(self.cluster_path)
+        worker_pattern = f'{cluster_id}-worker*'
+        worker_instances = self.aws.get_instances_by_name_pattern(
+            worker_pattern
+        )
+        security_groups = worker_instances[0]['security_groups']
```

Comment:
Is setting this in first worker sufficient?

Reply:
All the workers are using the same security group so I am taking SG just from first one worker.